po/sk_SK.po

```diff
@@ -18,187 +18,243 @@ msgstr ""
 "X-Generator: Poedit 3.0\n"
 "Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n>=2 && n<=4 ? 1 : 2);\n"
 
-#: data/input-remapper.glade:1128
-msgid "."
-msgstr "."
+#: inputremapper/gui/user_interface.py:612
+msgid ", CTRL + DEL to stop"
+msgstr ", zastavíte stlačením CTRL + DEL"
 
-#: data/input-remapper.glade:1348
-msgid "1, 2"
-msgstr "1, 2"
-
-#: data/input-remapper.glade:926
-msgid ""
-"A \"key + key + ... + key\" syntax can be used to trigger key combinations. "
-"For example \"control_l + a\".\n"
-"\n"
-"\"disable\" disables a key."
-msgstr ""
-"Syntax \"kláves + kláves + ... + kláves\" môžete použiť na zadávanie "
-"kombinácie klávesov. Napríklad \"control_l + a\".\n"
-"\n"
-"\"disable\" deaktivuje mapovanie klávesu."
-
-#: data/input-remapper.glade:898
+#: data/input-remapper.glade:1070
 msgid "About"
 msgstr "O programe"
 
-#: data/input-remapper.glade:209
+#: inputremapper/gui/user_interface.py:609
+#, python-format
+msgid "Applied preset %s"
+msgstr "Použité prednastavenie %1"
```

Comment:
`%1`, shouldn't this be `%s`?

Reply:
correct. force-pushed together with a few other issues.